client/components/plans/plans-compare/index.jsx

```diff
@@ -168,5 +181,23 @@ module.exports = React.createClass( {
 			</div>
 		);
 	}
-
 } );
+
+module.exports = connect(
+	( state, props ) => {
+		if ( props.isInSignup ) {
```

Comment:
Is relying on this prop still the best solution here? Couldn't we do something like looking at the user's current sites, and if they don't have any assume that it's in signup? I guess the problem with that is when users sign up for subsequent sites. I still feel like this prop isn't a great solution though.